vembrane/ann_types.py

```diff
@@ -92,55 +53,111 @@ def __init__(self, pos: int, length: int):
         self.length = length
 
     @classmethod
-    def from_str(cls, value: str) -> "PosLength":
-        pos, length = type_int_pair(value)
+    def from_snpeff_str(cls, value: str) -> "PosLength":
+        pos, length = [int(v.strip()) for v in value.split("/")]
         return cls(pos, length)
 
+    @classmethod
+    def from_vep_str(cls, value: str) -> "PosLength":
+        start, end = (
+            #  the "-" is optional, so vep either has either start and end position
+            [int(v.strip()) for v in value.split("-")]
+            if "-" in value
+            #  or start position only
+            else [int(value.strip())] * 2
+        )
+        return cls(start, start + end)
```

Comment:
Isn't the second value of the tumple the length and shouldn't it be calculated by `end - start`?

Reply:
... whooops